transforms/angle-brackets/transform.js

```diff
@@ -478,19 +491,10 @@ module.exports = function transform(fileInfo, config) {
 
     ElementNode(node) {
       node.attributes.forEach(a => {
-        if (a.value && a.value.chars === '') {
+        if (a.value && a.value.chars === '' && a.value.chars === _EMPTY_STRING_) {
```

Comment:
I don't fully understand this conditional. Either `a.value.chars` is `''` or it is the value of `_EMPTY_STRING` (which is `ANGLE_BRACKET_EMPTY_${Date.now()}`), it can't be both. AFAICT this branch will never be taken.

Reply:
I think this was a workaround between versions of recast(?).  I'll remove it and see what happens.   Empty attrs was giving me a headache that day. LOL